backend/python/tapa/verilog/xilinx/module.py

```diff
@@ -88,8 +88,12 @@ def _calculate_indices(self) -> None:
       elif isinstance(item, (ast.Assign, ast.Always)):
         self._last_logic_idx = idx
         if isinstance(item, ast.Assign):
-          if item.left.var.name in HANDSHAKE_OUTPUT_PORTS:
-            self._handshake_output_ports[item.left.var.name] = item
+          if isinstance(item.left, ast.Lvalue):
+            _name = item.left.var.name
```

Comment:
We don't use leading underscores in variable names. Also, when would the left-hand side of an `ast.Assign` be `ast.Identifier`? I thought the `ast.Identifier` is always wrapped in an `Lvalue`?

Reply:
Not sure about this, but will error out in the original way